examples/classification_and_regression.ipynb

```diff
@@ -0,0 +1,816 @@
+      },
+      "source": [
+        "network = ClassifierLocalizer(model_name='resnet18')"
+      ],
+      "execution_count": 10,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8482vAprYWgI"
+      },
+      "source": [
+        "## Loss function"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "DAa7rSqC3VJs"
+      },
+      "source": [
+        "As we discussed before, we have two different tasks in this example. These tasks need different loss functions; Cross-Entropy loss for the classification and Mean Square Error loss for the regression. Below, we define a new loss function class that sums both losses to considers them simultaneously. However, as the regression is relatively a simpler task here (due to similarity of coordinates in the images), we apply a lower weight to MSEloss."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "oCrU_k18J2Ps"
+      },
+      "source": [
+        "class ClassificationRegressionLoss(nn.Module):\n",
+        "    def __init__(self, W):\n",
+        "        super(ClassificationRegressionLoss, self).__init__()\n",
+        "        self.ce_loss = nn.CrossEntropyLoss() # size_average=False\n",
+        "        self.mse_loss = nn.MSELoss()\n",
+        "        self.W = W\n",
+        "        \n",
+        "    def forward(self, y_pred, y_true):\n",
+        "        #print(y_true[0][:,20])\n",
+        "        loss_cls = self.ce_loss(y_pred[0], y_true[0][:,20]) # Cross Entropy Error (for classification)\n",
+        "        loss_reg1 = self.mse_loss(y_pred[1][:,0], y_true[1][:,0]/h) # Mean Squared Error for X_L\n",
+        "        loss_reg2 = self.mse_loss(y_pred[1][:,1], y_true[1][:,1]/w) # Mean Squared Error for Y_L\n",
+        "        loss_reg3 = self.mse_loss(y_pred[1][:,2], y_true[1][:,2]/h) # Mean Squared Error for X_R\n",
+        "        loss_reg4 = self.mse_loss(y_pred[1][:,3], y_true[1][:,3]/w) # Mean Squared Error for Y_R\n",
+        "        total_loss = loss_cls + self.W * (loss_reg1 + loss_reg2 + loss_reg3 + loss_reg4)\n",
+        "        return total_loss"
+      ],
+      "execution_count": 11,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "e-ujQ315X9ku"
+      },
+      "source": [
+        "## Training"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "IqGF7KaxcgI3",
+        "outputId": "ac43c4ba-a626-4b44-da87-174d14b79b1d"
+      },
+      "source": [
+        "optimizer = optim.SGD(network.parameters(), lr=learning_rate, weight_decay=0.001)\n",
+        "loss_function = ClassificationRegressionLoss(W)\n",
+        "exp = Experiment('./saves/two_task_example', network, optimizer=optimizer, loss_function=loss_function, device=\"all\")\n",
```

Comment:
```suggestion
        "exp = Experiment('./saves/two_task_example', network, optimizer=optimizer, loss_function=loss_function, device=device)\n",
```
Use device instead of "all" since you defined device above.